hotels.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>SEDONA Hotels</title>
+</head>
+
+<body>
+    <header>
+        <h1 class="visualli-hidden">SEDONA</h1>
+        <figure class="sedona-logo">
+            <img src="images/logo.png" alt="logo">
+        </figure>
+        <nav class="sedona-navigation">
+            <ul>
+                <li class="sedona-navigation_item"><a href="#">информация</a></li>
+                <li class="sedona-navigation_item"><a href="#">фото и видио</a></li>
+                <li class="sedona-navigation_item"><a href="#">карта штата</a></li>
+                <li class="sedona-navigation_item"><a href="#">гостиницы</a></li>
+            </ul>
+        </nav>
+    </header>
+    <aside class="sedona-hotels-choose">
+
+        <form action="get">
+            <section class="sedona-hotels_check">
+                <h2 class="sedona-hotels_check_heading">инфраструктура</h2>
+
+                <input type="checkbox" name="subscribe-field-swimming-pool" id="subscribe-field-swimming-pool" checked>
+                <label for="subscribe-field-swimming-pool">бассейн</label>
+
+                <input type="checkbox" name="subscribe-field-parking" id="subscribe-field-parking">
+                <label for="subscribe-field-parking">парковка</label>
+
+                <input type="checkbox" name="subscribe-field-wi-fi" id="subscribe-field-wi-fi">
+                <label for="subscribe-field-wi-fi">wi-fi</label>
+            </section>
+
+            <section class="sedona-hotels_check">
+                <h2 class="sedona-hotels_check_heading">тип жилья</h2>
+
+                <input type="checkbox" name="subscribe-hausing-hotel" id="subscribe-hausing-hotel" checked>
+                <label for="subscribe-hausing-hotel">гостиница</label>
+
+                <input type="checkbox" name="subscribe-hausing-motel" id="subscribe-hausing-motel" checked>
+                <label for="subscribe-hausing-motel">мотель</label>
+
+                <input type="checkbox" name="subscribe-hausing-apartments" id="subscribe-hausing-apartments" checked>
+                <label for="subscribe-hausing-apartments">апартаменты</label>
+            </section>
+
+            <section class="sedona-hotels_check">
```

Comment:
rewrite `section` to `div`.
Sections aren't containers for elements of form. 
If you wanna wrap smth use div for it.